pkg/kv/kvserver/store_rangefeed.go

```diff
@@ -12,10 +12,41 @@ package kvserver
 
 import (
 	"context"
+	"math"
 	"time"
 
+	"github.com/cockroachdb/cockroach/pkg/clusterversion"
+	"github.com/cockroachdb/cockroach/pkg/kv/kvpb"
 	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/closedts"
+	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed"
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/settings"
 	"github.com/cockroachdb/cockroach/pkg/settings/cluster"
+	"github.com/cockroachdb/cockroach/pkg/util/limit"
+	"github.com/cockroachdb/cockroach/pkg/util/log"
+	"github.com/cockroachdb/cockroach/pkg/util/stop"
+	"github.com/cockroachdb/cockroach/pkg/util/syncutil"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"github.com/cockroachdb/errors"
+)
+
+var rangefeedRestartInterval = settings.RegisterDurationSetting(
+	settings.SystemOnly,
+	"kv.rangefeed.restarter.interval",
+	"minimum interval between rangefeed restarts to conform to new settings",
```

Comment:
Consider mentioning the setting that triggers this. Also, should we interpret 0 here as disabled, or will that restart with no delay?